docs/src/main/asciidoc/logging.adoc

```diff
@@ -56,24 +58,22 @@ public class ExampleResource {
 ----
 
 NOTE: While JBoss Logging routes log messages into JBoss Log Manager directly, one of your libraries might rely on a different logging API.
-In such cases, you need to use a <<logging-adapters,logging adapter>> to ensure that its log messages are routed to JBoss Log Manager as well.
+In such cases, you need to use a <<logging-apis,logging APIs>> to ensure that its log messages are routed to JBoss Log Manager as well.
```

Comment:
This is factually incorrect, logging APIs are already used (by those libraries). You may not like the term "logging adapter" (I actually think it's fairly good and precise), but we need some term to describe what's going on here.

Reply:
Fixed. It's just that I changed the section title. I've extended it to mention adapters too.